internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -97,8 +98,12 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		err     error
 	)
 	if f.ManagerConfigPath != "" {
-		cfgLoader := ctrl.ConfigFile().AtPath(f.ManagerConfigPath)
-		if options, err = options.AndFrom(cfgLoader); err != nil {
+		// TODO: option to load from config file is deprecated. This will also be removed from here when
+		// componentConfig option is removed.
+		//
+		// Refer: https://github.com/kubernetes-sigs/controller-runtime/issues/895
```

Comment:
Create an issue for following this up?

Reply:
Captured here: https://github.com/operator-framework/ansible-operator-plugins/issues/40